appcenter-crashes/scripts/postlink.js

```diff
@@ -1,29 +1,13 @@
 const rnpmlink = require('appcenter-link-scripts');
-const inquirer = require('inquirer');
 
 // Configure Android first.
 let promise = null;
 if (rnpmlink.android.checkIfAndroidDirectoryExists()) {
     console.log('Configuring AppCenter Crashes for Android');
     promise = rnpmlink.android.initAppCenterConfig()
-        .then(() =>
-            inquirer.prompt([{
-                type: 'list',
-                name: 'whenToSendCrashes',
-                message: 'For the Android app, should crashes be sent automatically or processed in JavaScript before being sent?',
-                choices: [
-                    {
-                        name: 'Automatically',
-                        value: 'ALWAYS_SEND'
-                    },
-                    {
-                        name: 'Processed in JavaScript by user',
-                        value: 'ASK_JAVASCRIPT'
-                    }]
-            }])
-        ).then((androidAnswer) => {
+        .then(() => {
             rnpmlink.android.patchStrings('appCenterCrashes_whenToSendCrashes',
-                androidAnswer.whenToSendCrashes);
+                'ALWAYS_SEND');
```

Comment:
This will overwrite the setting that will have to be manually changed by customer.
PatchStrings must be updated to only add the default value the first time.